pkg/util/pretty/prefix_printer.go

```diff
@@ -5,8 +5,9 @@ import "github.com/pterm/pterm"
 // Pretty prefix printer style.
 //
 // Usage:
-//   pretty.Info.Println("Hello")
-//   pretty.Warning.Println("Hello")
+//
```

Comment:
double check: https://pkg.go.dev/kusionstack.io/kusion@v0.7.2/pkg/util/pretty

![image](https://user-images.githubusercontent.com/23924124/206347301-36d6396c-1cc7-497c-a9c2-5d7f055d60be.png)

Reply:
Same format issue. It is recognized as blocker format issue in golangci-lint v1.50.1.